src/board/BoardTitle.tsx

```diff
@@ -50,9 +50,15 @@ export default function BoardTitle(props: BoardTitleProps) {
       })}
     >
       <HighlightedText highlightColor={props.accentColor || "#fffff"}>
-        <span className="title-text">{props.title}</span>
+        <div className="title-text">{props.title}</div>
       </HighlightedText>
       {titleStyles}
+      <style jsx>{`
```

Comment:
I'm ok with this change, but just FYI you should be able to obtain the same result by adding `display: inline-block;` or `display: block;` to the `span`, without changing it to a `div`. In this case, however, both work!